smart_proxy_pulp.gemspec

```diff
@@ -18,5 +18,5 @@ Gem::Specification.new do |gem|
   gem.require_paths = ["lib"]
   gem.license = 'GPL-3.0'
 
-  gem.required_ruby_version = '~> 2.5'
+  gem.required_ruby_version = '>= 2.5'
```

Comment:
This is weird. When I look at https://ci.theforeman.org/blue/organizations/jenkins/smart_proxy_pulp-pr-test/detail/smart_proxy_pulp-pr-test/3/pipeline it runs the tests on Ruby 3.0 & 3.1, but this shouldn't work. And yet it does pass. Something's wrong.

Reply:
In `bundle install` I don't see `smart_proxy_pulp` show up:
```
Using /usr/local/rvm/gems/ruby-3.1.2 with gemset smart_proxy_pulp-pr-test-3-1
Fetching https://github.com/theforeman/smart-proxy
Fetching gem metadata from https://rubygems.org/.........
Resolving dependencies...
Fetching rake 13.1.0
Installing rake 13.1.0
Fetching builder 3.2.4
Fetching rexml 3.2.6
Fetching public_suffix 5.0.4
Fetching ast 2.4.2
Installing builder 3.2.4
Installing ast 2.4.2
Installing rexml 3.2.6
Installing public_suffix 5.0.4
Fetching power_assert 2.0.3
Installing power_assert 2.0.3
Fetching hashdiff 1.1.0
Fetching json 2.7.1
Installing hashdiff 1.1.0
Installing json 2.7.1 with native extensions
Fetching little-plugger 1.1.4
Installing little-plugger 1.1.4
Fetching multi_json 1.15.0
Installing multi_json 1.15.0
Fetching mocha 1.16.1
Fetching racc 1.7.3
Fetching parallel 1.24.0
Installing mocha 1.16.1
Installing parallel 1.24.0
Installing racc 1.7.3 with native extensions
Fetching rack 2.2.8
Installing rack 2.2.8
Fetching rainbow 3.1.1
Fetching regexp_parser 2.8.3
Installing rainbow 3.1.1
Installing regexp_parser 2.8.3
Fetching ruby-progressbar 1.13.0
Installing ruby-progressbar 1.13.0
Fetching unicode-display_width 2.5.0
Installing unicode-display_width 2.5.0
Fetching sd_notify 0.1.1
Installing sd_notify 0.1.1
Fetching tilt 2.3.0
Installing tilt 2.3.0
Fetching webrick 1.8.1
Installing webrick 1.8.1
Fetching addressable 2.8.6
Fetching test-unit 3.6.1
Installing addressable 2.8.6
Installing test-unit 3.6.1
Fetching ci_reporter 2.1.0
Installing ci_reporter 2.1.0
Fetching crack 0.4.5
Installing crack 0.4.5
Fetching mustermann 2.0.2
Fetching logging 2.3.1
Installing mustermann 2.0.2
Installing logging 2.3.1
Fetching rack-protection 2.2.4
Installing rack-protection 2.2.4
Fetching rack-test 2.1.0
Installing rack-test 2.1.0
Fetching ci_reporter_test_unit 1.0.1
Installing ci_reporter_test_unit 1.0.1
Fetching webmock 3.19.1
Installing webmock 3.19.1
Fetching sinatra 2.2.4
Installing sinatra 2.2.4
Fetching parser 3.2.2.4
Installing parser 3.2.2.4
Fetching rubocop-ast 1.30.0
Installing rubocop-ast 1.30.0
Fetching rubocop 1.28.2
Installing rubocop 1.28.2
Fetching rubocop-performance 1.19.1
Fetching rubocop-rake 0.6.0
Installing rubocop-rake 0.6.0
Installing rubocop-performance 1.19.1
Bundle complete! 11 Gemfile dependencies, 41 gems now installed.
Gems in the group 'development' were not installed.
Use `bundle info [gemname]` to see where a bundled gem is installed.
```

Yet in the clone step is is cloning the right repository. What am I missing?

Reply:
The run nr 3 was with my patch

https://ci.theforeman.org/blue/organizations/jenkins/smart_proxy_pulp-pr-test/detail/smart_proxy_pulp-pr-test/1/pipeline did fail as expected

Reply:
See, I was missing something ...

Can you separate out the change? Then I can merge that immediately and we can just rely on Jenkins for now while we have the discussion on how to organize Smart Proxy plugin CI.

Reply:
I'm not at the laptop today anymore, so feel free to do yourself or we'll continue Monday.

Reply:
https://github.com/theforeman/smart_proxy_pulp/pull/37